deployment/build.debian.all

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+#= Debian/Ubuntu all .deb
+
+set -o errexit
+set -o xtrace
+
+# Move to source directory
+pushd ${SOURCE_DIR}
+
+cp -a yarn.lock /tmp/yarn.lock
+
+# Build DEB
+dpkg-buildpackage -us -uc --pre-clean --post-clean
+
+mkdir -p ${ARTIFACT_DIR}/
+mv ../jellyfin[-_]* ${ARTIFACT_DIR}/
```

Comment:
This command moved even cloned repo directory :smile:
```suggestion
mv ../jellyfin[-_]*.* ${ARTIFACT_DIR}/
```
but still not ideal

Reply:
Yea this one is a bit of a PITA. I could always do `jellyfin*.{deb,dsc,tar,gz,deb,buildinfo,changes}` to catch the 5 main files we want.